test/graphConfig.js

```diff
@@ -2,25 +2,78 @@ var GraphConfig = require('../lib/graphConfig');
 var test = require('tape');
 var fs = require('fs');
 var path = require('path');
+var assign = require('lodash.assign');
 var properties = require('properties');
 var nconf = require('nconf');
 
-nconf.file('base', __dirname + '/mocks/prod.json');
-nconf.file('prod', __dirname + '/mocks/base.json');
+var formats = {
+    '.properties' : {
+        parse: function(data) {
+            return properties.parse(data, {
+                'namespaces': true
+            });
+        },
+        stringify: properties.stringify
+    }
+};
+
+function initConfigLayers(opts) {
```

Comment:
Agree. I didn't like it when I wrote it. It actually duplicates function from `bin/config-builder.js`. I need to pull that functionality into its own module so that we can just reference it from this test, which will feel much simpler.

Will do that when I get time, but given this is just code for the test, I'm going to say we're ok. Will open a new issue to track this, though.